components/ui/Button.tsx

```diff
@@ -66,6 +67,7 @@ export const LinkButton = ({
   children,
   ...props
 }) => {
+  const id = sanitizeLabel(children[0].toString());
```

Comment:
Instead of using the children which can be anything - instead can we pass down some unique value (if thats whats intended)